<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>644</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>cortfly</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/image.qrc">
    <normaloff>:/main.png</normaloff>:/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/skin3.png);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/1.png);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="nichnameButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="images/image.qrc">
          <normaloff>:/my.JPG</normaloff>:/my.JPG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>74</width>
          <height>76</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/background1.png);</string>
          </property>
          <property name="text">
           <string>本机信息：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblWelcome">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/background1.png);</string>
        </property>
        <property name="text">
         <string>局域网在线：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listUsers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>连接服务器</string>
          </property>
          <property name="text">
           <string>实验室服务器</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>3</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>连接实验室论坛</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/background1.png);</string>
          </property>
          <property name="text">
           <string>实验室论坛</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionJoin">
   <property name="text">
    <string>Join</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="actionMe">
   <property name="text">
    <string>me</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/image.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onMenu(QAction*)</slot>
 </slots>
</ui>
